source/scss/hefe.scss

```diff
@@ -0,0 +1,22 @@
+/*
+ *
+ * Hefe
+ *
+ */
+@import 'shared';
+
+%hefe,
+.hefe {
+  @extend %filter-base;
+  filter: brightness(1.1) contrast(1.1) sepia(.1) saturate(1.15);
+
+  &::after {
+    box-shadow: inset .3rem -.3rem 20px rgba(21, 12, 0, .7);
+  }
+
+  &::before {
+    background: rgba(67, 45, 31, .7);
+    mix-blend-mode: overlay;
+    box-shadow: inset -.3rem .3rem 20px rgba(21, 12, 0, .7);
```

Comment:
Is it ok to use em?  I haven't figured how to use % in box-shadow and em is responsive too :)